cfme/tests/automate/test_method.py

```diff
@@ -4,12 +4,23 @@
 
 from cfme import test_requirements
 from cfme.automate.explorer.klass import ClassDetailsView
+from cfme.automate.simulation import simulate
 from cfme.utils.appliance.implementations.ui import navigate_to
 from cfme.utils.update import update
 
 pytestmark = [test_requirements.automate]
 
 
+@pytest.fixture(scope='function')
+def original_class(domain):
+    # Take the 'Request' class and copy it for own purpose.
+    domain.appliance.collections.domains.instantiate(name="ManageIQ").namespaces.instantiate(
```

Comment:
If `domain` is an entity of `Domains` collection, you could use `domain.parent.instantiate`, instead of `domain.appliance.collection.instantiate`

Reply:
Awesome suggestion to improvise code!
Thanks @valaparthvi